### DISCUSSION TITLE
Welcome to dbatools Discussions!

### DISCUSSION THREAD

**Discussion by potatoqualitee** [dataplat/dbatools#7006] [General] [answered]
## 👋 Welcome!
We’re using Discussions as a place to connect with other members of our community. We see it being like StackOverflow, but nice and welcoming!

We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jpomfret** [comment, marked as answer]
Very cool!

**Reply by potatoqualitee** [reply]
That's the answer! 😆

**Comment by sanderstad** [comment]
This is awesome

**Reply by LeonardRutkowski** [reply]
love this tool

**Comment by LeonardRutkowski** [comment]
Is there a way, to use the same login, to various dbatools commands.  Currently, I am using the -Credentials option for all of my commands. I l create the credential then do -Credential $Credential, as an example. But, I noticed, I am getting multiple logins to the SQL instance. I am executing a loop, to add logins, and each command seems to do a separate login, but doesn't disconnect, so now I have multiple logins, when I really only need to login and use a single connection.  I am doing other commands, as well, and so each one connects as a separate connection. 

Thanks, 
Leonard

**Reply by LeonardRutkowski** [reply]
Wrong thread. Apologies
